mautic/mautic
# Fix Hubspot processing batch limit

(cherry picked from commit ede3856c6ccba429fed42dffd269171b2182b46b)



| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Based on docs https://legacydocs.hubspot.com/docs/methods/contacts/get_contacts 
should  we use "count" 

> &count=X - Used in the request URL

![image](https://github.com/mautic/mautic/assets/462477/b0866b62-f547-4029-8049-52360165eadd)




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Just code review
